assembl/static2/js/app/utils/utilityManager.js

```diff
@@ -0,0 +1,37 @@
+/*
+  Singleton object that will contain the AlertManager, ModalManager which will be used to show/hide/manipulate the alert/modal system
+*/
+class UtilityManager {
+  setComponent(comp) {
+    this.component = comp;
+  }
+};
+
+export const modalManager = new UtilityManager();
+
+export const alertManager = new UtilityManager();
+
+export const displayAlert = (style, msg, topPosition = false, time = 6000) => {
```

Comment:
I'm okay with this, _for now_, but I wanted to write methods in the future such as `displayError`, `displaySuccess`, etc etc. This will start to make the file feel polluted.